### PR TITLE
Add windows install instructions

### PR DESCRIPTION
**Pull request by ibrahiminfinite**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,7 +21,13 @@ The first command above will install pytorch-cpu, which, as the name suggests,
  does not have cuda support. 
  
 `Note: inltk is currently supported only on Linux with Python >= 3.6`
- 
+
+### Intallation on windows (experimental)
+```bash
+pip3 install https://download.pytorch.org/whl/cpu/torch-1.1.0-cp36-cp36m-win_amd64.whl
+pip3 install torchvision
```
Comment (goru001) [inline README.md]: @ibrahiminfinite Is there any specific reason we need to install torchvision? I think it should work without it as well.

Comment (ibrahiminfinite) [reply README.md]: @goru001 its been removed.

**Comment by goru001** [comment]
@ibrahiminfinite You've tested this on Windows 10, is it?

**Comment by ibrahiminfinite** [comment]
Yes, tested on windows 10, in conda environment python 3.6,  with just pytorch and inltk installed.

**Pull request merged**